css/style.css

```diff
@@ -0,0 +1,34 @@
+#image {
+  display: block;
+  margin: 0 auto;
+}
+
+#img-caption {
+  font-size: 20px;
+  font-style: italic;
+}
+
+#img-div {
+  margin: 0;
+  text-align: center;
+  background-color: rgba(255, 255, 255, 0.452);
+}
+
+#main {
+  background-color: rgba(56, 56, 56, 0.575);
+}
+
+li {
+  margin-left: 20%;
+  margin-right: 20%;
+  padding-left: 20px;
+  font-style: italic;
+  font-size: 20px;
+  padding-bottom: 10px;
+}
```

Comment:
[OPTIONAL] Consider using the shorter version of doing margins and padding, it might be harder to understand but more used and practical to use!
```css
margin: top right bottom left
```